src/resources/Platforn/PlatformInterfaces.ts

```diff
@@ -0,0 +1,32 @@
+import {UpdateStatusCategory} from '../Enums';
+
+export interface MaestroVersionOptions {
+    crawlingModuleVersion: string;
+}
+
+export interface CrawlingModule {
+    createdDate: 0;
+    databaseVersion?: string;
+    disabled: true;
```

Comment:
```suggestion
    disabled: boolean;
```
Same for everywhere there is a `true`